src/app/modules/OpenshiftStreams/components/StreamsTableConnected/StreamsTableConnected.tsx

```diff
@@ -362,10 +362,15 @@ export const StreamsTableConnected: FunctionComponent<StreamsTableProps> = ({
     setSearchParam,
   ]);
 
+  const mounted = useRef(false);
+
   useEffect(() => {
-    setKafkaDataLoaded(false);
-    fetchKafkas();
-  }, [auth, page, perPage, filteredValue, orderBy, fetchKafkas]);
```

Comment:
I think we should not remove dependencies `[page, perPage, filteredValue, orderBy]` , it will stop calling fetchKafkas function to fetch new data on sorting , searching and on page change

Reply:
👍